pkg/utils/config.go

```diff
@@ -17,24 +17,27 @@ import (
 
```

Comment:
can we join config with defaults.go ?

Reply:
We plan to merge also with cmd/defaults.go, but in next PRs